pom.xml

```diff
@@ -150,6 +150,44 @@ limitations under the License.
           </configuration>
         </plugin>
 
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <executions>
+            <!-- Disable enforcer rules inherited from google-cloud-shared-config
+            and customize the rules so that they fit hbase deps -->
+            <execution>
+              <id>enforce</id>
+              <phase>none</phase>
+            </execution>
+            <execution>
+              <id>enforce-cbt</id>
+              <goals>
+                <goal>enforce</goal>
+              </goals>
+              <configuration>
+                <rules>
+                  <requireMavenVersion>
+                    <version>[3.0,)</version>
+                  </requireMavenVersion>
+                  <requireJavaVersion>
+                    <version>[1.8,)</version>
+                  </requireJavaVersion>
+                  <requireUpperBoundDeps/>
+                  <banDuplicateClasses>
+                    <scopes>
+                      <scope>compile</scope>
+                      <!-- remove provided scope from google-cloud-shared -->
```

Comment:
can you reword this - took me a minute. Maybe: "override from google-cloud-shared-config to only check for compile deps"

Reply:
done